opennms-base-assembly/src/main/filtered/etc/events/opennms.default.threshold.events.xml

```diff
@@ -2,110 +2,59 @@
    <event>
       <uei>uei.opennms.org/threshold/highThresholdExceeded</uei>
       <event-label>OpenNMS-defined threshold event: highThresholdExceeded</event-label>
-      <descr>&lt;p>High threshold exceeded for %service% datasource
-            %parm[ds]% on interface %interface%, parms: %parm[all]%&lt;/p>
-            &lt;p>By default, OpenNMS watches some key parameters
-            on devices in your network and will alert you with
-            an event if certain conditions arise. For example, if
-            the CPU utilization on your Cisco router maintains an
-            inordinately high percentage of utilization for an extended
-            period, an event will be generated. These thresholds are
-            determined and configured based on vendor recommendations,
-            tempered with real-world experience in working
-            deployments.&lt;/p> &lt;p>This specific event
-            indicates that a high threshold was exceeded.&lt;/p></descr>
+      <descr>A threshold alarm for the following metric occured: %parm[all]%</descr>
```

Comment:
I would prefer to not speak from an "alarm" here and keep it more specific as it was originally, e.g. like this:

> <descr>High threshold exceeded for the metric %parm[all]%</descr>

Reply:
fixed